src/component/renderable/schema/question/Question.ts

```diff
@@ -17,15 +17,16 @@ export default class Question {
   readonly options: IFormMakerOptions;
   readonly _parts: Array<QuestionPart>;
   readonly button: HTMLButtonElement;
-  private _removeButton: HTMLButtonElement;
-  private _addButton: HTMLButtonElement;
+  public _removeButton: HTMLButtonElement;
+  public _addButton: HTMLButtonElement;
```

Comment:
public field must be in Camel-Case without _ in start!
```suggestion
  public removeButton: HTMLButtonElement;
  public addButton: HTMLButtonElement;
```